modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -786,7 +932,7 @@ FeatureFloodCount::visitElementalNeighbors(const Elem * elem, unsigned long curr
   std::vector<const Elem *> all_active_neighbors;
 
   // Loop over all neighbors (at the the same level as the current element)
-  for (unsigned int i = 0; i < elem->n_neighbors(); ++i)
+  for (decltype(elem->n_neighbors()) i = 0; i < elem->n_neighbors(); ++i)
```

Comment:
You more consistently used `for (auto i = decltype(elem->n_neighbors())(0)...` throughout the rest of the code.  Might want to change this one to match.